Dockerfile.redhat

```diff
@@ -198,9 +198,10 @@ RUN wget -nv https://github.com/linux-test-project/lcov/releases/download/v1.16/
 COPY check_coverage.bat /ovms/
 
 ARG CHECK_COVERAGE=0
+ARG RUN_TESTS=1
 # Test Coverage
 RUN if [ "$CHECK_COVERAGE" = "1" ] ; then bazel coverage --combined_report=lcov --test_summary=detailed --test_output=streamed //src:ovms_test > ${TEST_LOG} 2>&1 || { cat ${TEST_LOG} && rm -rf ${TEST_LOG} && exit 1 ; } && genhtml --output genhtml "$(bazel info output_path)/_coverage/_coverage_report.dat" ; fi ; \
-    bazel test ${debug_bazel_flags} --jobs=$JOBS --test_summary=detailed --test_output=streamed //src:ovms_test > ${TEST_LOG} 2>&1 || (cat ${TEST_LOG} && rm -rf ${TEST_LOG} && exit 1 ; ) && tail -n 100 ${TEST_LOG} && rm -rf ${TEST_LOG}
+    if [ "$RUN_TESTS" == "1" ] ; then bazel test ${debug_bazel_flags} --jobs=$JOBS --test_summary=detailed --test_output=streamed //src:ovms_test > ${TEST_LOG} 2>&1 || (cat ${TEST_LOG} && rm -rf ${TEST_LOG} && exit 1 ; ) && tail -n 100 ${TEST_LOG} && rm -rf ${TEST_LOG} ; fi ;
```

Comment:
`make docker_build RUN_TESTS=0 CHECK_COVERAGE=1` still runs unit tests. Do we want it like that? Maybe give an error when using `RUN_TESTS=0 CHECK_COVERAGE=1`? You cannot check coverage without running tests.

Reply:
I think this way its is less complicated. Making additional checks and flags will not be readable. If user wants to run tests he sets the flag. If he doesn't want to run anything he sets two flags.

Reply:
Ok it works with 1 flag at the beginning so its ok.